src/classes/PS_IntegrationService_TEST.cls

```diff
@@ -35,16 +35,112 @@
 * @description Test class for the PS_IntegrationService class
 */
 
-@isTest
+@IsTest
 private class PS_IntegrationService_TEST {
-    
+
+    private static Map<String, List<Map<String, String>>> configParams;
+    private static Map<String, PS_IntegrationService.Config> makanaConfigParams;
+    private static Map<String, PS_IntegrationService.Config> paymentConfigParams;
+    private static final String API_KEY = 'apiKey';
+    private static final String PAYMENTS_APPLICATION_TYPE = 'payments';
+    private static final String MAKANA_APPLICATION_TYPE = 'makana';
+
+    static {
+        buildConfigurationParams(); // Build Test Configuration Parameters
+    }
+
     /*******************************************************************************************************
-     * @description tests storing of the secrets we receive from the Payment Services
-     * Integrations package
+     * @description Test for storing the secrets we receive from the Payment Services
+     * Integrations package with a valid API Key
      */
-    @isTest
-    private static void setConfigurationTest() {
-        // TODO This method can be implemented after the setConfiguration method is implemented
-         
+    @IsTest
+    private static void setConfigurationWithPaymentsAPIKey() {
```

Comment:
I believe the naming convention for our unit tests is `shouldDo...`, `shouldNotDo...`, etc so something like `shouldSetConfigurationWithPaymentsAPIKey`

Reply:
Updated.